source/software/hybrid_mpi_openmp_programs.rst

```diff
@@ -254,19 +254,20 @@ Essential elements of our job script are:
 
    #! /bin/bash -l
    # Adapt nodes and ppn on the next line according to the cluster your're using!
-   #PBS -lnodes=2:ppn=20
+   #PBS -l nodes=2:ppn=20
    ...
    #
    module load foss
    #
    export OMP_NUM_THREADS=5
    #
    export OMP_PROC_BIND=true
+   
+   MPI_NUM_PROCS=$(( $PBS_NP/$OMP_NUM_THREADS ))
    #
-   mpirun -cpus-per-proc $OMP_NUM_THREADS ./hybrid_mpi
+   mpirun --np $MPI_NUM_PROCS  --map-by socket:PE=$OMP_NUM_THREADS  --bind-to core  ./hybrid_mpi
```

Comment:
The (deprecated) ``-cpus-per-proc <num>`` option is still mentioned in the paragraph above -- could you also adapt that part?

Reply:
True, modified.